src/lp_data/Highs.cpp

```diff
@@ -2874,14 +2873,12 @@ HighsStatus Highs::callRunPostsolve(const HighsSolution& solution,
     // Set up the iteration count and timing records so that
     // adding the corresponding values after callSolveLp gives
     // difference
-    postsolve_iteration_count = -info_.simplex_iteration_count;
     timer_.start(timer_.solve_clock);
     call_status = callSolveLp(
         incumbent_lp,
         "Solving the original LP from the solution after postsolve");
     timer_.stop(timer_.solve_clock);
     // Determine the iteration count and timing records
-    postsolve_iteration_count += info_.simplex_iteration_count;
```

Comment:
Same comment here--it looks like removing this line makes part of the preceding comment obsolete.

Reply:
Ditto